layouts/partials/katex.html

```diff
@@ -0,0 +1,4 @@
+<!-- Based on https://mertbakir.gitlab.io/hugo/math-typesetting-in-hugo/ -->
+<link rel="stylesheet" href="/katex@0.16.7/katex.min.css" />

```

Comment:
This should have `rel="preload" as="style"` as per https://web.dev/defer-non-critical-css/

Reply:
Is this normal? https://katex.docs-cbr.pages.dev/guide/gamemodes/bhop/#technical-explanation

Reply:
Yeah it was the defer, fixed